### PR TITLE
Updated include guards to be KWIVER_ARROWS_* instead of KWIVER_ALGO_*

### PR DESCRIPTION
**Pull request by anthony-barricelli**
Updated include guards to be KWIVER_ARROWS_* instead of KWIVER_ALGO_* as per Issue #759

### CODE REVIEW

File: arrows/cuda/integrate_depth_maps.h [hunk 1]
```diff
@@ -95,4 +95,4 @@ class KWIVER_ALGO_CUDA_EXPORT integrate_depth_maps
 }  // end namespace arrows
 }  // end namespace kwiver
 
-#endif // KWIVER_ALGO_CUDA_INTEGRATE_DEPTH_MAPS_H_
+#endif // KWIVER_ALGO_ARROWS_INTEGRATE_DEPTH_MAPS_H_
```
Comment (mleotta) [inline arrows/cuda/integrate_depth_maps.h]: This one replaced "CUDA" instead of "ALGO"

**Comment by mleotta** [comment]
These changes look good, but it only covers a subset of the files that have this problem.  Almost all of the arrows have classes that need to be updated in this way.  We could merge this branch as is, but additional changes are need to fully resolve #761

**Comment by anthony-barricelli** [comment]
I see, I wasn't sure if we just wanted to update the include guards or the class names as well.

**Comment by mleotta** [comment]
I think the class names are okay, the included guards the primary focus of this issues, but feel free to point out class names that don't look consistent.

**Comment by anthony-barricelli** [comment]
Hopefully the last commit addresses what you meant.

**Comment by mleotta** [comment]
Ah, I see what you mean now by class names.  I didn't realize that most of those "KWIVER_ALGO_*" instances were in the "EXPORT" macros.  I think we do want to change the naming as you did here, expect I think we also need to change where those macros are created to match.  These are defined in the "*_export.h" header files that are generated by CMake and are based on the name of the libraries, which also needs to be updated (I was going to make a separate issue for that).

So I think the best course of action is to remove your last commit and put it on a new branch with a new PR.  We can merge you previous two commits as is.  On the new branch we also need to change the library names in CMake from "kwiver_algo_*" to "kwiver_arrows_*"

**Comment by mleotta** [comment]
Jenkins test this please

**Pull request merged**